h/security/permits/predicates.py

```diff
@@ -0,0 +1,140 @@
+from h.models.group import JoinableBy, ReadableBy, WriteableBy
+from h.security.permits.engine import predicate
+
+# Identity things
+
+
+@predicate()
+def authenticated(identity, context):
+    return bool(identity)
```

Comment:
Strictly speaking I think the `bool` is unnecessary as we do it in the engine anyway?

Reply:
It seems Pythonic for the engine to accept any truthy or falsey return value from theses predicate functions, rather than strictly requiring `True` or `False`. Truthiness or falsiness is what the engine actually needs, the exact type doesn't matter.